smartcontractkit/external-adapters-js
# Change unit & integration tests to use PR head instead of merge commit

## Description

Overriding the commit parent in the codecov action did not seem to work, changing unit & integration tests instead to run using the PR head. This shouldn't cause problems in theory when merging, because it's now enforced for merged PR branches to be up to date with the base branch.

## Changes

- Changes checkout action to pull PR head in unit & integration tests.

## Steps to Test

1. Github checkout action should now use the PR head commit as-is
2. Codecov coverage delta should not indicate anything unexpected

## Quality Assurance
